test/controllers/interrupt.card.controller.spec.ts

```diff
@@ -42,4 +74,30 @@ describe("INTERRUPT CARD controller", () => {
       expect(resp.text).toContain("Trusts");
     });
   });
+
+  describe("GET with url params tests", () => {
+    test(`renders the ${INTERRUPT_CARD_PAGE} page`, async () => {
+      mockIsActiveFeature.mockReturnValueOnce(true);
+
+      const resp = await request(app).get(INTERRUPT_CARD_WITH_PARAMS_URL);
+
+      expect(resp.status).toEqual(200);
+      expect(resp.text).toContain(INTERRUPT_CARD_PAGE_TITLE);
+      expect(resp.text).toContain(LANDING_PAGE_URL);
+      expect(resp.text).toContain(NEXT_PAGE_URL);
+      expect(mockGetUrlWithParamsToPath).toHaveBeenCalledTimes(1);
+      expect(mockGetUrlWithParamsToPath.mock.calls[0][0]).toEqual(OVERSEAS_NAME_WITH_PARAMS_URL);
+    });
+
+    test(`renders the ${INTERRUPT_CARD_PAGE} page with trust feature flag false`, async () => {
+      (isActiveFeature as jest.Mock).mockReturnValue(false);
```

Comment:
can (isActiveFeature as jest.Mock) be replaced with mockIsActiveFeature?

Reply:
Most probably. Will change.

Reply:
Done.